.gitpod.yml

```diff
@@ -56,11 +56,15 @@ tasks:
       cp .gitpod/default.local.services.yml docroot/sites/local.services.yml &&
       blt blt:telemetry:disable --no-interaction &&
       blt settings &&
+      rm -rf .git &&
       drush si stanford_profile -v -y &&
       drush cr &&
       drush cim -y &&
       drush then $THEME_NAME &&
-      drush cset system.theme default $THEME_NAME -y
+      drush cset system.theme default $THEME_NAME -y &&
+      drush sqlq "UPDATE config_pages__su_site_dropdowns SET su_site_dropdowns_value = 1 WHERE bundle = 'stanford_basic_site_settings'" &&
+      drush php-eval 'use  \Drupal\node\Entity\Node; $node = Node::load(13); $node->setPublished(true); $node->save();' &&
```

Comment:
This could be simpler without the `use`: `\Drupal\node\Entity\Node::load(13)->setPublished()->save();`